### ISSUE TITLE
User back-reference should not be translated

### ISSUE THREAD

**Issue by petterreinholdtsen** [lamby/django-auto-one-to-one#6]
I am testing the auto-one-to-one with the django.contrib.auth.models.User model, and a model like this:

```
class UserProfile(AutoOneToOneModel(User, 'profile')):
    uuid = models.UUIDField(unique = True,
                            default = uuid.uuid4,
                            editable = False)
    lastupdatetime = models.DateTimeField(_('When this entry was last updated'),
                                          auto_now = True)
```

When I try to filter on lastupdatetime to find users, I was very surprised to discovered I have to use code like this to point back to the user:

```
        for profile in UserProfile.objects.order_by('-lastupdatetime'):
            print(profile.brukar,
                  profile.brukar.date_joined,
                  profile.brukar.last_login)
```

I expected 'profile.user' to work, but intead I have to use the nn_NO name of the model, ie 'brukar' to refer back to the user model.  Can django-auto-one-to-one be changed to use the non-translated model name, and thus make sure the code work independent of the translation activated for the project?

**Comment by petterreinholdtsen** [comment]
Ah, forgot to mention that using 'attr' when setting up the UserProfile work, ie "class UserProfile(AutoOneToOneModel(User, 'profile', attr='user')):".  I just think it should be unnecessary.

**Comment by lamby** [comment]
Fixed in 3.1.1. Uploading to Debian now.
